docs/src/man/getting_started.md

```diff
@@ -111,61 +123,202 @@ julia> nulls(Int, 1, 3)
+│ 3   │ 3 │ F │
+│ 4   │ 4 │ M │
+│ 5   │ 5 │ F │
+│ 6   │ 6 │ M │
+│ 7   │ 7 │ M │
+│ 8   │ 8 │ F │
+
 ```
 
-The `DataFrame` we build in this way has 8 rows and 2 columns. You can check this using `size` function:
+The `DataFrame` we build in this way has 8 rows and 2 columns. You can check this using the
+`size`, `nrow`, and `ncol` functions:
+
+```jldocttest
+julia> using DataFrames
+
+julia> df = DataFrame()
+0×0 DataFrames.DataFrame
+
+
+julia> df[:A] = 1:8;
+
+julia> df[:B] = ["M", "F", "F", "M", "F", "M", "M", "F"];
+
+julia> size(df, 1) == nrow(df) == 8
+true
+
+julia> size(df, 2) == ncol(df) == 2
+true
+
+julia> size(df) == (8, 2)
+true
 
-```julia
-nrows = size(df, 1)
-ncols = size(df, 2)
 ```
 
 We can also look at small subsets of the data in a couple of different ways:
 
-```julia
-head(df)
-tail(df)
+```jldocttest
+julia> using DataFrames
+
+julia> df = DataFrame(A = 1:8, B = ["M", "F", "F", "M", "F", "M", "M", "F"]);
+
+julia> head(df)
+6×2 DataFrames.DataFrame
+│ Row │ A │ B │
+├─────┼───┼───┤
+│ 1   │ 1 │ M │
+│ 2   │ 2 │ F │
+│ 3   │ 3 │ F │
+│ 4   │ 4 │ M │
+│ 5   │ 5 │ F │
+│ 6   │ 6 │ M │
+
+julia> tail(df)
+6×2 DataFrames.DataFrame
+│ Row │ A │ B │
+├─────┼───┼───┤
+│ 1   │ 3 │ F │
+│ 2   │ 4 │ M │
+│ 3   │ 5 │ F │
+│ 4   │ 6 │ M │
+│ 5   │ 7 │ M │
+│ 6   │ 8 │ F │
+
+julia> df[1:3, :]
+3×2 DataFrames.DataFrame
+│ Row │ A │ B │
+├─────┼───┼───┤
+│ 1   │ 1 │ M │
+│ 2   │ 2 │ F │
+│ 3   │ 3 │ F │
 
-df[1:3, :]
 ```
 
 Having seen what some of the rows look like, we can try to summarize the entire data set using `describe`:
 
-```julia
-describe(df)
+```jldoctest
+julia> using DataFrames
+
+julia> df = DataFrame(A = 1:8, B = ["M", "F", "F", "M", "F", "M", "M", "F"]);
+
+julia> describe(df)
+A
+Summary Stats:
+Mean:           4.500000
+Minimum:        1.000000
+1st Quartile:   2.750000
+Median:         4.500000
+3rd Quartile:   6.250000
+Maximum:        8.000000
+Length:         8
+Type:           Int64
+
+B
+Summary Stats:
+Length:         8
+Type:           String
+Number Unique:  2
+
+
 ```
 
 To focus our search, we start looking at just the means and medians of specific columns. In the example below, we use numeric indexing to access the columns of the `DataFrame`:
 
-```julia
-mean(Nulls.skip(df[1]))
-median(Nulls.skip(df[1]))
+```jldoctest
+julia> using DataFrames
+
+julia> df = DataFrame(A = 1:8, B = ["M", "F", "F", "M", "F", "M", "M", "F"]);
+
+julia> mean(df[1])
+4.5
+
+julia> median(df[1])
+4.5
+
 ```
 
 We could also have used column names to access individual columns:
 
-```julia
-mean(Nulls.skip(df[:A]))
-median(Nulls.skip(df[:A]))
+```jldoctest
+julia> using DataFrames
+
+julia> df = DataFrame(A = 1:8, B = ["M", "F", "F", "M", "F", "M", "M", "F"]);
+
+julia> mean(df[:A])
+4.5
+
+julia> median(df[:A])
+4.5
+
 ```
 
 We can also apply a function to each column of a `DataFrame` with the `colwise` function. For example:
 
-```julia
-df = DataFrame(A = 1:4, B = randn(4))
-colwise(c->cumsum(Nulls.skip(c)), df)
```

Comment:
That example was quite weird: better apply a summary statistic like the mean or sum.